packages/shorebird_cli/lib/src/artifact_builder.dart

```diff
@@ -277,30 +279,50 @@ class ArtifactBuilder {
         ...args,
       ];
 
-      final result = await process.run(
+      final buildProcess = await process.start(
         executable,
         arguments,
         runInShell: true,
         environment: base64PublicKey?.toPublicKeyEnv(),
       );
 
-      if (result.exitCode != ExitCode.success.code) {
-        throw ArtifactBuildException('Failed to build: ${result.stderr}');
+      final stdoutLines = <String>[];
+      buildProcess.stdout
+          .transform(utf8.decoder)
+          .transform(const LineSplitter())
+          .listen((line) {
+        stdoutLines.add(line);
+        if (buildProgress == null) {
+          return;
+        }
+
+        final update = _progressUpdateFromIpaBuildLog(line);
+        if (update != null) {
+          buildProgress.updateDetailMessage(update);
+        }
+      });
+
+      final stderrLines = await buildProcess.stderr
+          .transform(utf8.decoder)
+          .transform(const LineSplitter())
+          .toList();
+      final stdErr = stderrLines.join('\n');
```

Comment:
stderr vs. stdErr?  I don't feel strongly, just looks like you used it sometimes as stdErr and sometimes stderr.

Reply:
Fixed